### PR TITLE
Update ubuntu18 to latest version

### PR DESCRIPTION
**Pull request by ChriMarMe**
Signed-off-by: Christopher Meis <christopher.meis@9elements.com>

### CODE REVIEW

File: operating-system/ubuntu/Ubuntu18.yaml [hunk 1]
```diff
@@ -59,6 +64,15 @@ actions:
     chroot: true
     script: debos_enable_ssh.sh
 
+# Install newest kernel
+  - action: apt
+    description: Install some more packages
```
Comment (jensdrenhaus) [inline operating-system/ubuntu/Ubuntu18.yaml:69]: description: Update kernel

File: operating-system/ubuntu/Ubuntu18.yaml [hunk 2]
```diff
@@ -59,6 +64,15 @@ actions:
...
+    packages:
+      - linux-image-5.4.0-51-generic 
+      - linux-modules-5.4.0-51-generic
+      - linux-firmware
```
Comment (jensdrenhaus) [inline operating-system/ubuntu/Ubuntu18.yaml:73]: Is this independent from updating bionic or focal?
And independent to what "latest" will be sometime?

File: operating-system/ubuntu/update_ubuntu.sh [hunk 3]
```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
```
Comment (jensdrenhaus) [inline operating-system/ubuntu/update_ubuntu.sh:1]: rename to `debos_update_ubuntu.sh`

**Review by jensdrenhaus — CHANGES_REQUESTED** [review]
If the update-script parsed the argument (18, 20), doesn't the Ubuntu20.yaml need an update to invoke this script, too?

File: operating-system/ubuntu/Ubuntu18.yaml (hunk 2 above)
Comment (ChriMarMe) [reply operating-system/ubuntu/Ubuntu18.yaml:73]: This is only for bionic. Kernel versions need to be revised now and then to get the latest kernel version.

File: operating-system/ubuntu/Ubuntu18.yaml (hunk 1 above)
Comment (ChriMarMe) [reply operating-system/ubuntu/Ubuntu18.yaml]: done

File: operating-system/ubuntu/update_ubuntu.sh (hunk 3 above)
Comment (ChriMarMe) [reply operating-system/ubuntu/update_ubuntu.sh]: done

File: operating-system/ubuntu/Ubuntu18.yaml [hunk 4]
```diff
@@ -44,6 +44,11 @@ actions:
     description: Overlay - Add files to system
     source: debos-overlays
 
+  - action: run
+    description: Update Ubuntu to latest
+    chroot: true
+    script: update_ubuntu.sh 18
```
Comment (jensdrenhaus) [inline operating-system/ubuntu/Ubuntu18.yaml:50]: `debos_update_ubuntu.sh 18`

**Review by jensdrenhaus — CHANGES_REQUESTED** [review]

File: operating-system/ubuntu/Ubuntu18.yaml (hunk 4 above)
Comment (ChriMarMe) [reply operating-system/ubuntu/Ubuntu18.yaml]: done

**Review by jensdrenhaus — APPROVED** [review]

**Pull request merged**